libs/domain/user/address-form/address-form.component.spec.ts

```diff
@@ -276,26 +296,94 @@ describe('UserAddressFormComponent', () => {
         expect.arrayContaining([
           expect.objectContaining({ id: 'isDefaultShipping' }),
         ]),
-        undefined
+        {}
       );
     });
+
+    describe('and there is an address available with is set to default shipping', () => {
+      beforeEach(async () => {
+        addressService.getAddresses.mockReturnValue(
+          of([{ isDefaultShipping: true } as Address])
+        );
+        element = await fixture(html` <oryx-user-address-form
+          enableDefaultShipping
+        ></oryx-user-address-form>`);
+      });
+
+      it('should set isDefaultShipping to true in the default values', () => {
+        expect(renderer.buildForm).toHaveBeenCalledWith(
+          expect.anything(),
+          expect.not.objectContaining({ isDefaultShipping: true })
```

Comment:
Might be a mistake. Should set isDefaultShipping to true, but expects that it is not